data/templates/bootstrap.js

```diff
@@ -79,33 +78,22 @@ function updateTasks(data, timeout, retries) {
                 }, timeout);
             } else {
                 console.log("Task Execution Complete");
-
                 process.exit(data.exit.code || 0);
             }
         });
     }).on('error', function (err) {
-        console.log("Update Tasks Error: " + err);
-
-        if (retries === undefined) {
-            retries = 1;
-        } else {
-            retries = retries + 1;
-        }
-
-        if (retries < RETRIES) {
+            console.log("Update Tasks Error: " + err);
+            if (retries === undefined){
+                retries = 1;
+            }else {
+                retries = retries + 1;
+            }
             console.log("Retrying Update Tasks Attempt #" + retries);
 
             setTimeout(function () {
-                updateTasks(data, timeout, retries);
+                updateTasks(data, timeout, retry, retries);
             }, timeout * retries);
```

Comment:
@DavidjohnBlodgett Now that you are allowing for an infinite number of retries should you still scale this up by the number of retries without some kind of upper limit?